lib/view/datatable/DataTable.php

```diff
@@ -89,35 +89,37 @@ public function __construct($orm, $dataUrl, $titel = false, $groupByColumn = nul
 			'defaultContent' => ''
 		);
 
-		if (is_a($orm, CustomDataTableEntry::class, true)) {
-			foreach ($orm::getFieldNames() as $attribute) {
-				$this->addColumn($attribute);
-			}
+		if ($includeAllColumns) {
```

Comment:
Misschien `includeColumns` noemen, of deze logica naar een methode `loadColumns` trekken die dan in een subclass ge-override kan worden.

Reply:
Heb nu beide gedaan. Voor dit vind ik een boolean in de constructor iets cleaner